keyvi/tests/keyvi/index/internal/merge_job_test.cpp

```diff
@@ -24,7 +24,9 @@
  */
 
 #include <chrono>  // NOLINT
+#include <string>
 #include <thread>  // NOLINT
+#include <vector>
```

Comment:
same here

Reply:
same answer: `string` and `vector` are used in this file, however this file hasn't been touched since the introduction of cpplint as CI workflow.